lighthouse-treemap/app/src/main.js

```diff
@@ -33,18 +33,15 @@ class TreemapViewer {
    * @param {HTMLElement} el
    */
   constructor(options, el) {
-    const treemapDebugData = /** @type {LH.Audit.Details.DebugData} */ (
+    const scriptTreemapData = /** @type {LH.Audit.Details.TreemapData} */ (
       options.lhr.audits['script-treemap-data'].details);
```

Comment:
```suggestion
    const scriptTreemapData = options.lhr.audits['script-treemap-data'].details;
```

Reply:
ah, the problem went away with the `dom` change.